### ISSUE TITLE
EDDI 3.0.1-b2 source won't compile

### ISSUE THREAD

**Issue by rodan123** [EDCD/EDDI#545]
Downloaded the source for EDDI 3.0.1-b2 from the releases tab.</br>
Won't compile cleanly in VS2017. Any Ideas?</br>
EDDI 3.0.1-b1 source compiles without errors.</br>
</br>
Also, In the develop branch:</br>
Commit **Convert logo to PNG  #1714299**</br>
File **Installer.iss** still references the original bitmap file.

**Comment by rodan123** [comment]
Not using code from a git branch.

Downloaded from https://github.com/EDCD/EDDI/archive/Release/3.0.1-b2.zip and unzipped to its own folder. Won't compile.
If I do the same from https://github.com/EDCD/EDDI/archive/Release/3.0.1-b1.zip there are no issues and I get a clean compilation.

I tried the command you suggested on the develop branch in git. Still can't compile that. Don't know why, obviously you guys are compiling successfully. Will keep looking at it. Let me know if you have any other suggestions.

Thanks,

**Comment by Tkael** [comment]
Are you using a git command line tool? Git Bash (https://gitforwindows.org/) works pretty well for me when I need to do something from the command line.

**Comment by rodan123** [comment]
Yep. Using git from the command line. Haven't bothered with the git GUI tools.

Not understanding why the source from the project download page won't compile though. That should be a clean source tree of the current beta release and not require any manipulation using git.

**Comment by Tkael** [comment]
While VerticalBlank and I didn't have trouble compiling b2 from the source, Hoodathunk did run into some trouble similar to what you are describing. We're still not sure why... reinstalling the package doesn't appear to create any change in the repo but it it does seem to help with this issue.
